src/Calculator/Views/CalculatorProgrammerRadixOperators.xaml.cpp

```diff
@@ -47,7 +47,12 @@ void CalculatorProgrammerRadixOperators::FlyoutButton_Clicked(_In_ Platform::Obj
 
 void CalculatorProgrammerRadixOperators::checkDefaultBitShift()
 {
-    this->ArithmeticShiftButton->IsChecked = true;
+    CollapseBitshiftButtons();
+    m_selectedShiftButtonMode = BitShiftMode::Arithmetic;
+    LshButton->Visibility = ::Visibility::Visible;
+    RshButton->Visibility = ::Visibility::Visible;
+    LshButton->IsEnabled = true;
+    RshButton->IsEnabled = true;
 }
```

Comment:
In previous versions, there was an issue where button availability was reset when opening BitShift Flyout not when the mode is changed